homeassistant/components/uptimerobot/switch.py

```diff
@@ -18,6 +18,9 @@
 from .coordinator import UptimeRobotConfigEntry, UptimeRobotDataUpdateCoordinator
 from .entity import UptimeRobotEntity
 
+# Limit the number of parallel updates to 1
+PARALLEL_UPDATES = 1
```

Comment:
What is the rationale behind limiting this to 1?

Reply:
Because if you set it to more than 1 and do parallel updates, you get an API error.
At least this is what I experienced during my tests.